tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -46,8 +46,10 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
         Try(IronOxide.userCreate(jwt, primaryTestUserPassword, new UserCreateOpts(true), defaultTimeout)).toEither
       val createResult = resp.value
 
+      createResult.equals(createResult) shouldBe true
```

Comment:
All of the asserts to self don't make much sense here, as they would be true even if we didn't do hashCode and equals overrides.

Reply:
All the tests for this have been redone